packages/renderer/src/lib/kube/details/KubeContainerPorts.spec.ts

```diff
@@ -0,0 +1,65 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import '@testing-library/jest-dom/vitest';
+
+import { render } from '@testing-library/svelte';
+import { expect, test } from 'vitest';
+
+import KubeContainerPorts from '/@/lib/kube/details/KubeContainerPorts.svelte';
+
+test('expect port title not to be visible when no ports provided', async () => {
+  const { queryByText } = render(KubeContainerPorts);
+
+  const title = queryByText('Ports');
+  expect(title).toBeNull();
+});
+
+test('expect port title to be visible when ports is defined', async () => {
+  const { getByText } = render(KubeContainerPorts, {
+    ports: [
+      {
+        containerPort: 80,
+      },
+    ],
+  });
+
+  const title = getByText('Ports');
+  expect(title).toBeDefined();
+});
+
+test('expect multiple ports to be visible', async () => {
```

Comment:
If the point of the PR was to switch them to a column, maybe we should test parent to confirm flex-col?